src/sections/Pricing/index.js

```diff
@@ -42,58 +42,101 @@ const getCustomToggleButtonStyle = (isActive, baseStyle) => ({
 });
 
 export const CurrencySelect = ({ currency, setCurrency }) => {
+  const [open, setOpen] = useState(false);
+
+  // Lock both <html> and <body> while the dropdown is open
+  useEffect(() => {
+    if (!open) return;
```

Comment:
this logic seems bit low level. Why the `disableScrollLock` is not working? Try using modal component and see if its wokring without such logic to handle manually the scrolling